platform/ios/framework/strip-frameworks.sh

```diff
@@ -53,6 +53,18 @@ for file in $(find . -type f -perm +111); do
   if ! [[ "$(file "$file")" == *"dynamically linked shared library"* ]]; then
     continue
   fi
+
+  # Check for required Mapbox frameworks
+  framework=$(basename $file)
+
+  if [ "$framework" = "Mapbox" ]; then
+    found_mapbox_sdk=true
+  fi
+
+  if [ "$framework" = "MapboxMobileEvents" ]; then
```

Comment:
MapboxMobileEvents could be present but it could be an incompatible version. Perhaps check the framework bundle’s Info.plist for a version that matches the map SDK’s requirements (accounting for semver).

Incidentally, do we explicitly specify the supported or required MapboxNavigationNative versions anywhere in this repository? All I can find are references to “the latest version”, which could potentially become inaccurate.